cmd/backend/build_training_env/build-training-env.go

```diff
@@ -0,0 +1,78 @@
+package build_training_env
+
+/*
+$ your_cli_tool build-training-env --secret "YourTrainingSecretName" --bucket "YourTrainingBucketName"
+*/
+
+import (
+    "os"
+    "strings"
+    "fmt"
+    "log"
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/secretsmanager"
+    "github.com/spf13/cobra"
+    "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/backend"
+)
+
+var secretNameTraining string // Name of the secret in AWS Secrets Manager
+var bucketNameTraining string // Name of the bucket for training
+
+var buildTrainingEnvCmd = &cobra.Command{
+    Use:   "build-training-env",
+    Short: "Build .env file for training/",
+    Run: func(cmd *cobra.Command, args []string) {
+        sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
```

Comment:
can we make the `"us-west-2"` a constant since that region value is consistent across `dlp-cli`